tests/testData/default.yaml

```diff
@@ -0,0 +1,34 @@
+---
+# Default setting
+
+# Double value
+VAL_DOUBLE: 1.2
+
+# Integer value
+VAL_INTEGER: 2
+
+# Data distribution service (DDS) server telemetry port
+SERVER_PORT_TLM_DDS: 5570
+
+# Data distribution service (DDS) server command port
+SERVER_PORT_CMD_DDS: 5571
```

Comment:
Any chance of changing DDS to CSC now, in this common package? This low-level controller does not speak DDS at all! And we may switch away from DDS to some other messaging system, which will make this even more confusing in the future.

Reply:
I removed these in the commit: 0f7015f. I would like to redesign the existed codes in DM-33252. Thanks!